server/routes/user/index.ts

```diff
@@ -501,25 +501,21 @@ router.post(
       // taken from auth.ts
       const admin = await userRepository.findOneOrFail({
         where: { id: 1 },
-        select: [
-          'id',
-          'jellyfinAuthToken',
-          'jellyfinDeviceId',
-          'jellyfinUserId',
-        ],
+        select: ['id', 'jellyfinDeviceId', 'jellyfinUserId'],
         order: { id: 'ASC' },
       });
+
+      const hostname = getHostname();
       const jellyfinClient = new JellyfinAPI(
-        getHostname(),
-        admin.jellyfinAuthToken ?? '',
+        hostname,
```

Comment:
Why not inline it?

Reply:
Because the `hostname` variable was already used later in the code, so instead of replacing every reference to it i just moved it here

Reply:
Ah didn't see that there. Makes sense